composer.json

```diff
@@ -108,6 +106,7 @@
         "predis/predis": "^2.0.3",
         "psr/http-message": "^1.0",
         "psr/log": "^1.1",
+        "rulerz-php/doctrine-orm": "dev-master",
```

Comment:
added because wallabag rulerz packages are based on master from original packages, not the released 0.x ones.
no big changes, just that the support for databases was extracted in separate packages